client/store/productsReducer.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios'
+
+const LOAD_PRODUCTS = 'LOAD_PRODUCTS';
+
+const productReducers = (state = [], action) =>{
+    if(action.type === LOAD_PRODUCTS){
+        state = action.students;
+    }
+    return state;
+}
+
+const _loadProducts = (products) =>{
+    return {
+        type: LOAD_PRODUCTS,
+        products,
+    }
+
+}
+
+const loadProducts = () =>{
+    return async (dispatch) =>{
+        const products = (await axios.get('/api/products')).data
+        dispatch(_loadProducts(products))
+    }
+}
+
+export {loadProducts}
```

Comment:
with the 'export default' and 'export' included on lines 5 and 20, respectively, you can delete line 27